flink-container/pom.xml

```diff
@@ -0,0 +1,67 @@
+<!--
```

Comment:
Licenses headers in other pom files don't use `~`.

Reply:
Good catch. It slipped through. Will correct it.